tests/simulate/test_lammps.py

```diff
@@ -578,6 +616,39 @@ def test_writetrajectory(self):
             self.assertCountEqual(snap.typeid, [1, 1, 2, 2, 2])
             self.assertCountEqual(snap.mass, [1, 1, 1, 1, 1])
 
+    def test_writetrajectory_gsd(self):
```

Comment:
Can we simplify testing this operation by attaching both analyzers to the same simulation? Like, combine this test with what's above?

Reply:
We could! I had them separate so if they fail they show the descriptive name, but you could get that info from the traceback

Reply:
Yeah, I think the traceback is sufficient. The unittest framework will tell us which assertion is failing, so I would rather follow the traceback than have to maintain significant duplicate code.